### PR TITLE
pkg/logger: initial commit

### PR DESCRIPTION
**Pull request by gyuho**
Later to be used with "snapshot" and "clientv3" packages.

Separated from https://github.com/coreos/etcd/pull/9118.

### CODE REVIEW

**Comment by xiang90** [comment]
lgtm

File: pkg/logger/package_logger.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+// NewPackageLogger wraps "*capnslog.PackageLogger" that implements "Logger" interface.
+//
+// For example:
+//
+//  var defaultLogger Logger
+//  plog := capnslog.NewPackageLogger("github.com/coreos/etcd", "snapshot")
+//  defaultLogger = NewPackageLogger(plog)
+//
+func NewPackageLogger(p *capnslog.PackageLogger) Logger {
```
Comment (xiang90) [inline pkg/logger/package_logger.go]: probably do an assert that Logger satisfies grpc.Logger

Comment (xiang90) [reply pkg/logger/package_logger.go]: /cc @gyuho

Comment (xiang90) [reply pkg/logger/package_logger.go]: something like

```
var _ MyInterface = MyType{}
```

Comment (gyuho) [reply pkg/logger/package_logger.go]: Just added.

**Pull request merged**